packages/cli/src/lib/cmds/build.ts

```diff
@@ -55,6 +57,20 @@ class Build {
       private prompt: Prompt = new InquirerPrompt()) {
   }
 
+  /**
+   * Checks if the twa-manifest.json file has been changed since the last time the project was generated.
+   */
+  async checkManifest(manifestFile: string): Promise<boolean> {
+    const manifestContents = fs.readFileSync(manifestFile);
+    const checksumFile = path.join(process.cwd(), 'manifest-checksum.txt');
```

Comment:
We should put the checksum file in the same directory as the other files (`args.directory`).

Reply:
Changed it to use `args.directory` if it's there. Same thing in other parts of the code. The README should probably be updated to also have the `--directory` flag.